lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module

```diff
@@ -351,6 +351,20 @@ function dosomething_zendesk_verify_zendesk_user($email, $client = NULL) {
       // Create a zendesk user.
       return dosomething_zendesk_create_zendesk_user($email, $client);
     }
+    // Update user with profile fields if missing.
+    if (!$search->results[0]->user_fields->profile_url) {
```

Comment:
Just tested it and it worked fine but it did add an error to the log 

Notice: Undefined property: stdClass::$profile_url in dosomething_zendesk_verify_zendesk_user() 

Ill change it to isset and see what happens